src/Clock.elm

```diff
@@ -0,0 +1,103 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)

```

Comment:
The name of the module should always match the name of the file - in this case `module Clock exposing`